AirCAndC/AirCCPropertyListing.cs

```diff
@@ -0,0 +1,198 @@
+        }
+
+        /// <summary>
+        /// Sets square footage of property.
+        /// </summary>
+        /// <param name="squareFootage">Square footage.</param>
+        public void SetSquareFootage(int squareFootage)
+        {
+            this.squareFootage = squareFootage;
+        }
+
+        /// <summary>
+        /// Gets address of property.
+        /// </summary>
+        /// <returns>Address of property.</returns>
+        public string GetAddress()
+        {
+            return this.address;
+        }
+
+        /// <summary>
+        /// Sets address of property.
+        /// </summary>
+        /// <param name="address">Address of property.</param>
+        public void SetAddress(string address)
+        {
+            this.address = address;
+        }
+
+        /// <summary>
+        /// Gets the renting type of listing (daily, weekly, monthly).
+        /// </summary>
+        /// <returns>Renting package.</returns>
+        public RenterPackages GetRentTypeOffered()
+        {
+            return this.rentTypeOffered;
+        }
+
+
+        /// <summary>
+        /// Sets renting type of listing (daily, weekly, monthly).
+        /// </summary>
+        /// <param name="rentTypeOffered">Renting package.</param>
+        public void SetRentTypeOffered(RenterPackages rentTypeOffered)
+        {
+            this.rentTypeOffered = rentTypeOffered;
+        }
+
+        /// <summary>
+        /// Gets price associated with renting package chosen.
+        /// </summary>
+        /// <returns>Price of renting package.</returns>
+        public int GetPriceForPackage()
+        {
+            return this.priceForPackage;
+        }
+
+        /// <summary>
+        /// Sets price of renting package.
+        /// </summary>
+        /// <param name="priceForPackage">Price of renting package.</param>
+        public void SetPriceForPackage(int priceForPackage)
+        {
+            this.priceForPackage = priceForPackage;
+        }
+
+        /// <summary>
+        /// Gets list of amenities offered on property.
+        /// </summary>
+        /// <returns>Amenities offered.</returns>
+        public List<Amenities> GetAmenitiesOffered()
+        {
+            return this.amenities;
+        }
+
+        /// <summary>
+        /// Prints amenities associated with property.
+        /// </summary>
+        public void PrintAmenities()
+        {
+
+            foreach (Amenities amenities in this.amenities)
+            {
+                Console.WriteLine($" {amenities} ");
+            }
+            if (amenities.Count <= 0)
+            {
+                Console.WriteLine(" No amenities.");
+            }
+        }
+
+        /// <summary>
+        /// Sets amenities offered on property.
+        /// </summary>
+        /// <param name="amenities">Amenities offered.</param>
+        public void SetAmenitiesOffered(List<Amenities> amenities)
+        {
+            this.amenities = amenities;
```

Comment:
- [ ] Consider setting a clone of the list to the field to uphold Encapsulation.